### DISCUSSION TITLE
buddhism/2024/evening_ritual/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#129] [General]
# buddhism/2024/evening_ritual/

https://shanenull.com/buddhism/2024/evening_ritual/

**Comment by shane0** [comment]
# daily

- https://github.com/shane0/buddhism/issues/155 level 3 chapter notes

> “Waking up this morning, I smile. Twenty-four brand new hours are before me. I vow to live fully in each moment and to look at all beings with the eyes of love.”
>> [The Heart Of The Buddha's Teaching Hanh, Thich Nhat](https://github.com/shane0/buddhism/issues/162)

> The real world is simply too terrible to admit. It tells man that he is a small trembling animal who will someday decay and die. Culture changes all of this, makes man seem important, vital to the universe, immortal in some ways.
>> Ernest Becker

> humans are temporary pockets of low entropy
>> the universe - chapter 1 page 42

## chapter 16

```text
the nature of all things is neither real nor unreal
```

## greed is a waste of life

```text
If we have a hundred thousand, we want
a million. If we have a million, we want
ten million. Having ten million, we want a
billion; having a billion, we want the entire
country; having the entire country, we want
to conquer the world; having conquered the
world, we set our sights on outer space…
Human desire is endless and insatiable. By
attempting to satisfy it, our lives will be
immersed in greed, an ocean of suffering
from beginning to end.
```

- diamond sutra: subdue the mind and abide in nothing
 
## heart sutra

> the outside world unstable unpleasant and impersonal
>> everything is changing
>>> everything is atoms
>>>> atoms do not know. you exist therefore they cannot care if you exist or not
>>>>> your body is atoms they will rearrange when the universe decides not when you decide they are not yours
>>>>>> pure awareness

- stay out of the pure minds way there is no wisdom and no attainment
- abide in nothing and let go of everything so pure awareness is not hindered
- the heart sutra can be memorized and takes minutes
- chant it in the morning during the day and at night
- listen to the diamond sutra or prajnaparamita audio at night
- loop the audio so if you wake up during the night or in the morning you continue the practice

## death is the strongest motivation

- death is a reason to be charitable and grateful
- we are born with nothing and we take nothing with us but our karma

> anything that is changing is not real
>> anything conditional is changing
>>> anything that arises dependent on conditions is not real
>>>> anything that arises on conditions ceases

---

>  the motivation and reminder is maranasati 
>> the teaching is the heart sutra diamond sutra and mahaprajnaparamita
>>> value what goes with you when you die - good karma a warm heart 
>>>> the meditation is mind ground

---

- every day + hourly remember maranasati
- you will die one day prepare your mind
- kill your little ego before you die only the little ego has fear
- pure awareness is unborn and undying
- everything else is empty of independence and will cease
- we do not know when or how we die death does not care how healthy or young you are
- maranasati should not make you sad it should make you grateful
- when you wake up and you are aware you should be grateful and treat each day like it is the last because it could be
- when you go to bed you appreciate the sunset knowing it could be your last
- you sleep peacefully without fear of death because you might not wake up
- is motivation and a reminder that ultimately the money possessions doctors and drugs cannot save you 
- you cannot take anything with you but your good karma and a warm heart
- mind seal is practice for the trip

> seek nothing and obtain nothing
>> expect nothing

- heart sutra is the right view and reason letting go is easy there is nobody letting go and nothing to let go of not even wisdom or attainment
- heart sutra is memorized try to do it hourly silently if appropriate
- condensing the next memorization
- distilling the text in a command line app
- mind seal: recommends the diamond sutra every evening - 2x takes 40m?
- maranasati question is it the last sunrise or sunset
- oxherding can be used as a gauge
- https://gist.github.com/shane0/a9a62b532f0bbd0e091ceaa63cb73b0e#file-ten_oxherding-md

## fearless timeless suchness

> "I CANNOT ESCAPE DEATH. BUT AT LEAST I CAN ESCAPE THE FEAR OF IT." - EPICTETUS

- tibetans even meditate wakefully while they are asleep
- https://shanenull.com/buddhism/2021/meditation/dream-yoga/

**Reply by shane0** [reply]
# tldr

- stay out of the pure minds way

**Reply by shane0** [reply]
# sanskrit

https://shanenull.com/buddhism/2023/sanskrit/

**Reply by shane0** [reply]
### Pure and intrinsic awareness (jue-

That which knows and perceives, also known as the buddha nature inherent in all sentient beings.

**Reply by shane0** [reply]
# another 3 poisons

- do not accept reject or ignore anything https://youtu.be/akBc6oaRSxw

**Reply by shane0** [reply]
# good karma

- do all good
- perform no evil
- so simple a child understands it but an 80 year old hasn’t mastered it

## do

https://shanenull.com/buddhism/2024/six_paramitas/

## don’t

- three poisons greed hate ignorance
- ten unwholesome deeds

**Comment by shane0** [comment]
# basics

- pirmal confusion https://shanenull.com/buddhism/2023/confusion/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/samu/#samu-working-as-meditation

- samu click app

**Reply by shane0** [reply]
How To Write a Chinese Poem
88. A well-known Japanese poet was asked how to compose a Chinese poem.

"The usual Chinese poem is four lines," he explained. "The first line contains the initial phrase; the second line, the continuation of that phrase; the third line turns from this subject and begins a new one; and the fourth line brings the first three lines together. A popular Japanese song illustrates this:

Two daughters of a silk merchant live in Kyoto.
The elder is twenty, the younger, eighteen.
A soldier may kill with his sword,
But these girls slay men with their eyes."

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/draft/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/rivers/

**Reply by shane0** [reply]
https://shanenull.com/stoicism/3/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/leaks/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/eightfold/

**Reply by shane0** [reply]
https://shanenull.com/stoicism/3/

**Comment by shane0** [comment]
# previous daily meditations

- https://shanenull.com/buddhism/2021/docs_2021/sadhana/liberation/
- https://shanenull.com/buddhism/2021/meditation/vajrayana/
- https://shanenull.com/buddhism/2021/meditation/
- https://shanenull.com/buddhism/2023/walking/
- https://shanenull.com/buddhism/2023/record/
- https://shanenull.com/buddhism/latest/seven_full/
- https://shanenull.com/buddhism/2023/confusion/
- https://shanenull.com/buddhism/2021/meditation/dream-yoga/
- https://shanenull.com/buddhism/2023/oxherding/
- https://shanenull.com/buddhism/2021/docs_2021/sadhana/urgyen/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/eat/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/food/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/favorites/#what-are-phenomena-empty-of

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/six_paramitas/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/linji/

**Reply by shane0** [reply]
http://shanenull.com/buddhism/2021/docs_2021/sadhana/urgyen/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/quotes/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/#daily

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/anapanasati/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/eightfold_path/#diagram

**Reply by shane0** [reply]
# sadhana

- https://en.wikipedia.org/wiki/S%C4%81dhan%C4%81

**Comment by shane0** [comment]
# diamond sutra

- https://www.ctworld.org.tw/Buddhist%20e-Books/Books01/index.html
- https://github.com/shane0/buddhism/issues/92

> 32: All conditioned dharmas Are like a dream, an illusion, a bubble, a shadow, Like dew or a flash of lightning
>> ?? diamond sutra: subdue the mind and abide in nothing
>>> 28: recognize the egolessness of all things
>>>> 22: there is no Dharma to attain
>>>>> 21: there is no Dharma to teach

```text
All conditioned dharmas Are like a dream, an illusion, a bubble, a shadow, Like dew or a flash of lightning; Thus we shall perceive them. With this, the Buddha concluded the sutra. The elder Subhuti, other bhikshus, bhikshunis, upasakas, upasikas," heavenly and human beings, asuras and other beings of the world, having heard the Buddha, were all filled with immense joy; they accepted and followed the teaching faith-fully.
```

**Reply by shane0** [reply]
```text
All conditioned dharmas Are like a dream, an illusion, a bubble, a shadow, Like dew or a flash of lightning; Thus we shall perceive them.
```

**Reply by shane0** [reply]
# five skhandas

- form is like foam 
- feeling bubbles
- conception illusion
- volition plantain tree (hollow trunk)
- consciousness mirage

**Comment by shane0** [comment]
# SIGNS THAT OUR PRACTICE IS NOT GOING RIGHT

The more you practice, the more you do retreats, the more experiences you have, the more rituals you learn, and the more skills you acquire, if your ego becomes bigger and bigger and bigger in the process, that is an indication that your practice is not going right. Additionally, if you look down on others who have no idea about such practices, that is a further indication that your practice is not going right.

—Dzogchen Ponlop Rinpoche

**Reply by shane0** [reply]
There is no wisdom and no attainment. - heart sutra
there are no teachers and no teaching - diamond sutra

**Comment by shane0** [comment]
# chaconne

- https://youtu.be/zcGt9AFlIPY andres!

**Comment by shane0** [comment]
# meditation record or journal

- sometimes I create a google form for this purpose
- https://shanenull.com/buddhism/2023/record/

**Comment by shane0** [comment]
# signs it is going right

- https://youtu.be/7RwCtyAc2u4?si=5KcS6utXdRWkdQ8a
- you see and become the impermanence & emptiness
- you do not train your conscious mind to be non distracted that is just more fabrication

**Comment by shane0** [comment]
# prajna paramita

- https://youtube.com/playlist?list=PLJ1M78dFVbTyThaNlY0TFFAwB7cIMbccv&si=oHD8APrVO3fddusz

**Reply by shane0** [reply]
https://84000.co/translation/toh10

**Reply by shane0** [reply]
## summary chapters

```text
The Verse Summary of the Jewel Qualities (Toh 13)
```

```text
the story of Sadāprarudita and his guru Dharmodgata (chapters 85 and 86)
```

**Reply by shane0** [reply]
- https://84000.co/canon-sections/perfection-of-wisdom

**Comment by shane0** [comment]
# heart sutra

- https://84000.co/translation/toh21

In this famous scripture, known popularly as The Heart Sūtra, the Buddha Śākyamuni inspires his senior monk Śāriputra to request instructions from the bodhisattva Avalokiteśvara on the way to practice the perfection of wisdom. Avalokiteśvara then describes how an aspiring practitioner of the perfection of wisdom must first understand how all phenomena lack an intrinsic nature, which amounts to the realization of emptiness. Next, Avalokiteśvara reveals a brief mantra that the practitioner can recite as a method for engendering this understanding experientially. Following Avalokiteśvara’s teaching, the Buddha offers his endorsement and confirms that this is the foremost way to practice the perfection of wisdom.

```text

In this famous scripture, known popularly as The Heart Sūtra, the Buddha Śākyamuni inspires his senior monk Śāriputra to request instructions from the bodhisattva Avalokiteśvara on the way to practice the perfection of wisdom. Avalokiteśvara then describes how an aspiring practitioner of the perfection of wisdom must first understand how all phenomena lack an intrinsic nature, which amounts to the realization of emptiness. Next, Avalokiteśvara reveals a brief mantra that the practitioner can recite as a method for engendering this understanding experientially. Following Avalokiteśvara’s teaching, the Buddha offers his endorsement and confirms that this is the foremost way to practice the perfection of wisdom.
```

**Reply by shane0** [reply]
## Śāriputra
shA ri’i bu
ཤཱ་རིའི་བུ།
śāriputra
Also known as Śāradvatīputra. He was one of the closest disciples of the Buddha, known for his pure discipline and, of the hearer disciples, considered foremost in wisdom.

**Reply by shane0** [reply]
## Avalokiteśvara

spyan ras gzigs dbang phyug
སྤྱན་རས་གཟིགས་དབང་ཕྱུག
avalokiteśvara
One of the main bodhisattva disciples of the Buddha Śākyamuni, praised for his compassion.

**Reply by shane0** [reply]
https://github.com/shane0/buddhism/issues/104

**Comment by shane0** [comment]
# class notes

- update this site with my class notes
- https://shanenull.com/buddhism/2024/bujo/2024/

**Reply by shane0** [reply]
## six perfections

Dhyāna is defined as one-pointed abiding in an undistracted state of mind, free from afflicted mental states. Four states of dhyāna are identified as being conducive to birth within the form realm. In the context of the Mahāyāna, it is the fifth of the six perfections. It is commonly translated as “concentration,” “meditative concentration,” and so on.

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/six_paramitas/

**Reply by shane0** [reply]
charity
morality
tolerance
diligence
meditation
prajna wisdom

**Comment by shane0** [comment]
# fish

- https://www.dhammatalks.org/suttas/KN/StNp/StNp4_15.html
- When embraced, the rod of violence breeds danger & fear: Look at people quarreling. 
- I will tell of how I experienced dismay. 
- Seeing people floundering like fish in small puddles, competing with one another — as I saw this, fear came into me. 
- The world was entirely without substance. 
- All the directions were knocked out of line. 
- Wanting a haven for myself, I saw nothing that wasn't laid claim to. 
- Seeing nothing in the end but competition, I felt discontent. 
- And then I saw an arrow here, so very hard to see, embedded in the heart.
- Overcome by this arrow you run in all directions. But simply on pulling it out you don't run, you don't sink.
